data/mil2525d/core_data/gdbs/README.md

```diff
@@ -43,14 +43,19 @@ To view the Label Rules and associations, view the symbol dictionary table "Labe
 | ------------- | -------------- | --- | ----------------- | ----- |
 | additionalinformation  | H | H | Additional Information | |
 | additionalinformation2 | H | H | Additional Information 2 | Used in the Convoys and Airspace Coordination Area tactical graphics. |
```

Comment:
I'm wondering if we shouldn't also sort this table by Field ID (might be easier for folks using the standard)